importers/metadata.go

```diff
@@ -4,9 +4,10 @@ import "github.com/algorand/indexer/plugins"
 
 // ImporterMetadata returns fields relevant to identification and description of plugins.
 type ImporterMetadata struct {
```

Comment:
Thinking out loud. This might not be the right PR.

Maybe `PluginMetadata` should be a struct instead of an interface. Do we get anything from having an implementation of that interface in each plugin type? If they all shared the same type then you would only need to update one place and we could extend it without breaking the interface.

Reply:
I think it makes sense.  But perhaps another issue like you said.